.editorconfig

```diff
@@ -9,3 +9,12 @@ insert_final_newline = true
 max_line_length = 120
 tab_width = 4
 trim_trailing_whitespace = true
+
+[eclipse-codeStyle.xml]
+indent_size = unset
+indent_style = space
+
+[{integration-tests/src/test/resources/logback-test.xml,pig/src/main/resources/*.xml,pig/src/test/resources/*.xml}]
```

Comment:
@thescouser89  I know you prefer 4 spaces. Should these files just be converted to use 4 (instead of 2) spaces? The `eclipse-codeStyle.xml` is machine sgenerated and seems unindented, so it may be better to leave that one excluded.

Reply:
I do prefer 4 spaces.

Is that an ignore list? If so it looks fine.

Reply:
I need a wider monitor!!

Reply:
> I do prefer 4 spaces.
> 
> Is that an ignore list? If so it looks fine.

These patterns are set to 2 spaces instead of 4 (4 is the global option).

There are two options: leave integration-tests and pig using 2 and the rest use 4, or move everything to 4. If they're going to use 2, rather than add to the global `.editorconfig` like it is here, it may be better to place a new `.editorconfig` in certain directories with `*.xml`. This would allow pig to have different global formatting from the rest of bacon if it wants.

Reply:
> I need a wider monitor!!

You mean this line? There is no way to break it but you can come up with a more terse regex, or just use a separate file.

Too bad they basically got rid of 16:10 monitors, but they do have the 21:9 ultrawides now!

Reply:
> > I need a wider monitor!!
> 
> You mean this line? There is no way to break it but you can come up with a more terse regex, or just use a separate file.
> 
> Too bad they basically got rid of 16:10 monitors, but they do have the 21:9 ultrawides now!

Er, no,  sorry that was referring to the 4 spaces!

Reply:
> > I do prefer 4 spaces.
> > Is that an ignore list? If so it looks fine.
> 
> These patterns are set to 2 spaces instead of 4 (4 is the global option).
> 
> There are two options: leave integration-tests and pig using 2 and the rest use 4, or move everything to 4. If they're going to use 2, rather than add to the global `.editorconfig` like it is here, it may be better to place a new `.editorconfig` in certain directories with `*.xml`. This would allow pig to have different global formatting from the rest of bacon if it wants.

Why are some parts using different formatting?

Reply:
I assume because bacon and pig were two separate codebases and then became one.

Reply:
I think the formatting in the codebase should be consistent and changed.

Reply:
OK, so running `editorconfig:format` automatically fixes the formatting.

Reply:
Ok.